src/vs/editor/contrib/clipboard/clipboard.ts

```diff
@@ -142,6 +142,13 @@ class ExecCommandCopyAction extends ExecCommandAction {
 		if (!emptySelectionClipboard && editor.getSelection().isEmpty()) {
 			return;
 		}
+		// prevent copying an empty string by accident
+		if (editor.getSelections().length === 1 && editor.getSelection().isEmpty()) {
```

Comment:
You may want to compare between `editor.getModel().getLineFirstNonWhitespaceColumn` and `editor.getModel().getLineMaxColumn` instead of doing a substring.